### PR TITLE
fix(containerize): supports passing multiple tags

### PR DESCRIPTION
**Pull request by aarnphm**
## What does this PR address?

`-t` should create multiple docker image tag. This PR address the current
behaviour where the only last tag will be omitted.

## Before submitting:

- [ ] Does the Pull Request follow [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary) naming? Here are [GitHub's
      guide](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) on how to create a pull request.
- [ ] Does the code follow BentoML's code style, both `make format` and `make lint` script have passed ([instructions](https://github.com/bentoml/BentoML/blob/main/DEVELOPMENT.md#style-check-auto-formatting-type-checking))?
- [ ] Did you read through [contribution guidelines](https://github.com/bentoml/BentoML/blob/main/CONTRIBUTING.md#ways-to-contribute) and follow [development guidelines](https://github.com/bentoml/BentoML/blob/main/DEVELOPMENT.md#start-developing)?
- [ ] Did your changes require updates to the documentation? Have you updated
      those accordingly? Here are [documentation guidelines](https://github.com/bentoml/BentoML/tree/main/docs) and [tips on writting docs](https://github.com/bentoml/BentoML/tree/main/docs#writing-documentation).
- [ ] Did you write tests to cover your changes?

### CODE REVIEW

File: bentoml/_internal/utils/buildx.py [hunk 1]
```diff
@@ -165,19 +165,19 @@ def build(
     shm_size: str | int | None,
     rm: bool,
     ssh: str | None,
-    tags: str | list[str] | None,
+    tags: str | list[str],
```
Comment (sauyon) [inline bentoml/_internal/utils/buildx.py:168]:
```suggestion
    tags: str | t.Iterable[str],
```

File: bentoml/_internal/utils/buildx.py [hunk 2]
```diff
@@ -165,19 +165,19 @@ def build(
...
     cmds += ["--progress", progress]
 
-    if tags is None:
-        tags = []
-    tags = [tags] if not isinstance(tags, list) else tags
-    for tag in tags:
-        cmds.extend(["--tag", tag])
+    if isinstance(tags, str):
+        cmds.extend(["--tag", tags])
+    elif isinstance(tags, list) or isinstance(tags, tuple):
```
Comment (sauyon) [inline bentoml/_internal/utils/buildx.py:178]:
```suggestion
    elif isinstance(tags, t.Iterable):
```

File: bentoml/_internal/utils/docker.py [hunk 3]
```diff
@@ -67,3 +64,16 @@ def validate_tag(
             "or a dash and may contain a maximum of 128 characters."
         )
     return tag
+
+
+def validate_tag(
```
Comment (sauyon) [inline bentoml/_internal/utils/docker.py:69]: Do we need to validate docker tags? Does docker not do that for us?

File: bentoml/__init__.py [hunk 4]
```diff
@@ -1,3 +1,4 @@
+# pylint: skip-file
```
Comment (sauyon) [inline bentoml/__init__.py:1]: Why are we adding this now?

Comment (aarnphm) [reply bentoml/__init__.py:1]: otherwise pep8 bot would just throw warning.

Comment (sauyon) [reply bentoml/__init__.py:1]: Oh, is there not a pep8-specific one?

Comment (aarnphm) [reply bentoml/__init__.py:1]: pep8 just follow pylint warning iirc.

**Pull request merged**